### PR TITLE
Send Ethos K8s Operators (Kafka) logs to Splunk using fluent-bit sidecar

### PR DESCRIPTION
**Pull request by adapureddi-apparao**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| Related tickets | fixes #X, partially #Y, mentioned in #Z
| License         | Apache 2.0

### What's in this PR?
Centralizing and query the Kafka operator logs.
In this use-case, fluent-bit is used as sidecar and push the logs to Splunk server.

### Why?
Query and Analyze the Kafka operator logs.

### Additional context
Feature added to enable the LogForwarding to Splunk.
For enabling LogForwarding feature, set the logForward:create to true, then provide the Splunk and fluent-bit details.

### Checklist

- [ ] Implementation tested
- [ ] Error handling code meets the [guideline](https://github.com/banzaicloud/developer-guide/blob/master/docs/coding-style/error-handling-guide.md)
- [ ] Logging code meets the guideline
- [ ] User guide and development docs updated (if needed)

TEST RESULTS:
[Splunk](https://splunk.or1.adobe.net/en-US/app/search/report?s=%2FservicesNS%2Fadapured%2Fsearch%2Fsaved%2Fsearches%2Fpushinglogstosplunk&sid=1618997875.4914_DBEFF53A-73DA-4F86-8406-0515DFEE3168&dispatch.sample_ratio=1&display.page.search.mode=verbose&q=search%20index%3D%22plat_app_preprod%22%20pipeline-kafka-operator-operator-7898586788-k7b8f&earliest=-5m%40m&latest=now)

### CODE REVIEW

File: charts/kafka-operator/README.md [hunk 1]
```diff
@@ -82,3 +82,18 @@ Parameter | Description | Default
 `webhook.enabled` | Operator will activate the admission webhooks for custom resources | `true`
 `webhook.certs.generate` | Helm chart will generate cert for the webhook | `true`
 `webhook.certs.secret` | Helm chart will use the secret name applied here for the cert | `kafka-operator-serving-cert`
+`logForward:create` | If true, operator logs forward to Splunk | `false`
```
Comment (amuraru) [inline charts/kafka-operator/README.md:85]: stale? logForward and logSidecars are not used anymore

File: charts/kafka-operator/values.yaml [hunk 2]
```diff
@@ -72,3 +72,11 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+## Whether to create the LogForwarding or not.
+## For enabling LogForwarding feature, set the logForward.create to true.
+logForward:
```
Comment (amuraru) [inline charts/kafka-operator/values.yaml:78]: logForward as a parameter seems off here - can you remove it and use `additionalSidecars` instead to check?

**Comment by adapureddi-apparao** [comment]
Closing this PR and creating new PR

**Comment by adapureddi-apparao** [comment]
New PR:https://github.com/adobe/kafka-operator/pull/32

**Pull request closed without merging**